install_config/certificate_customization.adoc

```diff
@@ -248,7 +248,8 @@ ansible-playbook /usr/share/ansible/openshift-ansible/playbooks/redeploy-certifi
 ----
 
 [NOTE] 
-For this playbook to run, the certificate names must not change. If the certificate names change, rerun the Ansible *_deploy_cluster.yml_* playbook
```

Comment:
This part is irrelevant to customerized router certs, so no need this change.

Reply:
Thank you! Fixed.